### PR TITLE
refactor: integrate migrations package in styles package

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: .changeset/green-games-design.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': minor
+---
+
+Adds Migrations to Styles package.
```
Comment (gfellerph) [inline .changeset/green-games-design.md:5]:
```suggestion
Adds Angular migration Schematics to the @swisspost/design-system-styles package. These migrations will be automatically triggered when using `ng update @swisspost/design-system-styles` in your Angular project. For more information, have a look at the [migration guide](https://design-system.post.ch/).
```

File: packages/styles/README.md [hunk 2]
```diff
@@ -32,6 +32,30 @@ Available bundles:
 - basics.scss (atomic styles for font, buttons, lists etc.)
 - core.scss (variables, mixins, functions and placeholders only)
 
+## Migrations Schematics
+
+Schematics are used to transform a software project by adapting it to current Design System Styles.
```
Comment (gfellerph) [inline packages/styles/README.md:37]:
```suggestion
Schematics are used to transform an Angular software project by adapting it to current Design System Styles.
```

File: packages/styles/README.md [hunk 3]
```diff
@@ -32,6 +32,30 @@ Available bundles:
...
+
+Replace `migration-name` with any defined migration in the migrations.json file or simply get some help with:
+
+```bash
+schematics --help
+```
+
+### Migrate
+
+To execute the migrations please follow our [migration-guide](https://design-system.post.ch/).
```
Comment (gfellerph) [inline packages/styles/README.md:58]:
```suggestion
To execute the migrations please follow our [migration guide](https://design-system.post.ch/).
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

**Review by gfellerph — APPROVED** [review]

**Pull request merged**